sites/partners/src/components/listings/PaperListingForm/sections/ListingIntro.tsx

```diff
@@ -112,16 +160,40 @@ const ListingIntro = (props: ListingIntroProps) => {
               register={register}
               {...defaultFieldProps(
                 "developer",
-                enableHousingDeveloperOwner
-                  ? t("listings.housingDeveloperOwner")
-                  : t("listings.developer"),
+                developerFieldTitle,
                 props.requiredFields,
                 errors,
                 clearErrors
               )}
             />
           </Grid.Cell>
         </Grid.Row>
+        {listingType === EnumListingListingType.nonRegulated && enableNonRegulatedListings && (
+          <Grid.Row columns={1}>
+            <Grid.Cell>
+              <FieldGroup
```

Comment:
issue: When loading a listing with this value set on the listing the correct radio button isn't selected. I think it might just be because there is no `defaultChecked`

Steps to reproduce:
1. Save a listing with either "yes" or "no" selected for the "Has this property received HUD EBLL clearance?" question.
2. Exit the listing editing view
3. Click on "edit" for that listing.
4. Neither "no" or "yes" is selected